src/cargo-fmt/main.rs

```diff
@@ -53,7 +53,7 @@ pub struct Opts {
     #[structopt(name = "rustfmt_options", raw(true))]
     rustfmt_options: Vec<String>,
 
-    /// Format all packages (only usable in workspaces)
+    /// Format not only workspace packages, but also their local path-based dependencies
```

Comment:
IMO it'd probably be best to avoid the word `workspace` here to avoid potentially confusing folks with the impression this requires an explicit workspace like the original text did.

```suggestion
    /// Format all packages and their local path-based dependencies
```

I find the "all/and" combination a bit more explicit and clear, though I suppose a tweak to the "not only/but" combination could work too

```suggestion
    /// Format not only all packages, but also their local path-based dependencies
```

Reply:
`     /// Format all packages, and also their local path-based dependencies` perhaps?

Reply:
LGTM